css/main.css

```diff
@@ -70,6 +83,24 @@ ol {
   font-weight: 700;
   line-height: 25px;
 }
+.rectang-button:before {
```

Comment:
you don't need this before pseudo element.
you can just modify styles for button to create hover effect